### ISSUE TITLE
same device is shown multiple times in syncing device list

### ISSUE THREAD

**Issue by chaitanyaprem** [status-im/status-desktop#16240]
# Bug Report

## Description

Same device is shown more than once when synced multiple times. Ideally some sort of device-id can be used to identify that sync was done with same device and not show it again in the list. 

## Steps to reproduce

1. Install mobile app and sync with desktop instance successfully.
2. Uninstall mobile app and install again and sync with same desktop instance. 
3. Go to Settings -> Syncing and you will see same device shown as twice in sync devices list.

Note that the reason i had to uninstall mobile app is because i am dogfooding various version from different PR's and update always doesn't happen across them.

#### Expected behavior

A device even-though synced multiple times with a desktop instance should be shown only once in the device list. 

#### Actual behavior
Same device is shown multiple times in sync devices list. Here is screenshot of same where my mobile is shown 5 times in device list.
<img width="971" alt="Screenshot 2024-09-02 at 5 24 37 PM" src="https://github.com/user-attachments/assets/90a76de1-e3a3-41cc-9b2b-6f5943574d4f">

### Additional Information

- Status desktop version: https://github.com/status-im/status-desktop/commit/7c184f350
- Operating System: MacOS

**Comment by jrainville** [comment]
This might be an issue of naming or UX. Because this is  a feature and not an issue per se.

Each time you pair with a new device, you're not pairing with the device itself, you're pairing with that installation. So if you install, pair, remove, pair, you're pairing with two different installations, so it's two different ones.

I'm guessing we use the term `Device` in that screen because it's clearer to users what it is vs `Installation`, but I agree that in this case, it can be a bit confusing.

Let's ask designers what they think, but I wouldn't say it's a high priority issue, as it's kind of edge case.
cc @benjthayer @xAlisher

**Comment by caybro** [comment]
Could look sth like that:

![image](https://github.com/user-attachments/assets/e3ea415d-fa7e-47a3-bb59-caf2dda7d0b4)


CC @benjthayer

**Comment by jrainville** [comment]
Yeah that would work. The ID is quite bigger iirc though. Something like `0x1234567890-1234567890`

**Comment by caybro** [comment]
> Yeah that would work. The ID is quite bigger iirc though. Something like `0x1234567890-1234567890`

Yeah sure, I guess it will get elided automatically anyway

**Comment by caybro** [comment]
Yup: 

![image](https://github.com/user-attachments/assets/2fac840c-7922-4f73-a058-38e2e4fc7294)

**Comment by benjthayer** [comment]
> This might be an issue of naming or UX. Because this is a feature and not an issue per se.
> 
> Each time you pair with a new device, you're not pairing with the device itself, you're pairing with that installation. So if you install, pair, remove, pair, you're pairing with two different installations, so it's two different ones.
> 
> I'm guessing we use the term `Device` in that screen because it's clearer to users what it is vs `Installation`, but I agree that in this case, it can be a bit confusing.
> 
> Let's ask designers what they think, but I wouldn't say it's a high priority issue, as it's kind of edge case. cc @benjthayer @xAlisher

I.e. if I sync the same devices (albeit with a fresh install), can we somehow recognise that the 2 devices are already synced previously and remove the old synced device list item and replace it with the new one? 

Given a user will only be able to have one installation at a time on their device, I think they will care more about seeing this list reflect the devices that are synced than the installations. Sorry probably missing some details here but from a UX perspective I think that would be what the user would anticipate.

**Comment by jrainville** [comment]
> if I sync the same devices (albeit with a fresh install), can we somehow recognise that the 2 devices are already synced previously and remove the old synced device list item and replace it with the new one?

It's possible. I can investigate later, but it's a pretty low priority issue so I won,t have time soon to check

**Comment by jrainville** [comment]
I created [a new issue](https://github.com/status-im/status-desktop/issues/18409) to enable users to delete installations. 

I'll close this issue in favor of that new one, as I believe giving users the option is better than doing it in their stead as it could cause false negatives that would be unfortunate.
